### DISCUSSION TITLE
Help required

### DISCUSSION THREAD

**Discussion by CarDiaC001** [Anjok07/ultimatevocalremovergui#1571] [Q&A]
Hi just a quick question is there a plugin to solo the drums with this software?

**Comment by sdflemming** [comment]
Demucs Option (see attached image) will extract drums stem.
<img width="686" alt="Screenshot 2024-10-02 at 5 42 04 AM" src="https://github.com/user-attachments/assets/802319e8-2601-49d5-bac5-195356cdc5cb">
